platform_versioned_docs/version-23.4.0/enterprise/prerequisites/azure.mdx

```diff
@@ -7,182 +7,310 @@ tags: [azure, prerequisites, configuration]
+- **MySQL database**: An external database such as [Azure Database for MySQL][azure-db-create-portal] is highly recommended for production deployments.
 
-- **MySQL database**: An external database (i.e., external to your Docker Compose or Kubernetes deployment) is highly recommended for production deployments. If you don't have your own database service, you can use [Azure Database for MySQL](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal).
+- **SSL certificate**: An SSL certificate is required for your Seqera instance to handle HTTPS traffic. See [SSL/TLS configuration](../configuration/ssl_tls.mdx#azure-deployments-use-application-gateway-for-tls-termination-and-load-balancing) for more information.
 
-  If you decide to use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
+  :::caution
+  HTTP-only implementations **must** set the `TOWER_ENABLE_UNSAFE_MODE=true` environment variable in the Seqera hosting infrastructure to enable user login. HTTP must not be used in production environments.
+  :::
 
-:::note
-When creating a MySQL user, use the `USER@HOSTNAME` format for the `TOWER_DB_USER` environment variable. For Azure managed MySQL, it's [recommended](https://docs.microsoft.com/en-us/azure/mysql/connect-java#prepare-a-configuration-file-to-connect-to-azure-database-for-mysql) to pass an explicit `serverTimezone` to the `TOWER_DB_URL` environment variable, which (depending on your configuration) may be `UTC`. The connection string should therefore look like: `jdbc:mysql://MYSQL_INSTANCE_NAME.mysql.database.azure.com/TOWER_DATABASE?serverTimezone=UTC`.
-:::
-
-- **SSL certificate (optional)**: An SSL certificate is required for your Seqera instance to handle HTTPS traffic.
-
-From version 22.1.1, HTTP-only implementations **must** set the `TOWER_ENABLE_UNSAFE_MODE=true` environment variable in the Seqera hosting infrastructure to enable user login. HTTP must not be used in production environments.
-
-While there are many ways to implement DNS and TLS-termination, Seqera recommends using the specialized native services offered by your cloud provider. In the case of Azure:
+- **(Optional) DNS**: DNS is required to support human-readable domain names and load-balanced traffic. See [Azure DNS][azure-dns] to learn about domain aquisition and record management.
```

Comment:
The leading _optional_ kind of interrupts the parallelism. Not sure of a better way to do this, though. Have _optional_ after DNS? Start the sentence with _(Optional)_?